react/features/rtcstats/RTCStats.js

```diff
@@ -0,0 +1,78 @@
+import rtcstatsInit from 'rtcstats/rtcstats';
+import traceInit from 'rtcstats/trace-ws';
+
+/**
+ * Filter out RTCPeerConnection that are created by callstats.io.
+ *
+ * @param {*} config - Config object sent to the PC c'tor.
+ * @returns {boolean}
+ */
+function connectionFilter(config) {
+    if (config && config.iceServers[0] && config.iceServers[0].urls) {
+        for (const iceUrl of config.iceServers[0].urls) {
+            if (iceUrl.indexOf('taas.callstats.io') >= 0) {
+                return true;
+            }
+        }
+    }
+}
+
+/**
+ * Class that controls the rtcstats flow, because it overwrites and proxies global function it should only be
+ * initialized once.
+ */
+class RTCStats {
+    /**
+     * Initialize the rtcstats components. First off we initialize the trace, which is a wrapped websocket
+     * that does the actual communication with the server. Secondly, the rtcstats component is initialized,
+     * it overwrites GUM and PeerConnection global functions and adds a proxy over them used to capture stats.
+     * Note, lib-jitsi-meet takes references to these methods before initializing so the init method needs to be
+     * loaded before it does.
+     *
+     * @param {Object} options -.
+     * @param {string} options.rtcstatsEndpoint - The Amplitude app key required.
+     * @param {number} options.rtcstatsPollInterval - The getstats poll interval in ms.
+     * @returns {void}
+     */
+    init(options) {
+        this.trace = traceInit(options.rtcstatsEndpoint);
+        rtcstatsInit(this.trace, options.rtcstatsPollInterval, [ '', 'webkit', 'moz' ], connectionFilter);
```

Comment:
Do we need to pass the `webkit` and `moz` prefixes here ?
This is what the documentation of the rtcstats says
`When using ontop of adapter it is typically not necessary (and potentially harmful) to shim the webkit and moz prefixes in addition to the unprefixed version.`

Reply:
You're right, I think mozRTCPeerConnection/webkitRTCPeerConnection have been deprecated for some time now anyhow. Will remove and make sure it works properly.